### ISSUE TITLE
Rename things to be more distinct?

### ISSUE THREAD

**Issue by chrisranderson** [jart/tensorboard_plugin_example#2]
I'm struggling to figure out how importing works, and it would be nice if things didn't have the same name, or even similar names. Right now there is 

- `my_plugin` the directory
- `my-plugin` the path in the build rule
- `my-plugin` the component name
- `my-plugin.html`
- `my-plugin` the key to the `TENSORBOARD_PLUGINS` object 
- `my-plugin` the value to the `TENSORBOARD_PLUGINS` object 

I had a different naming scheme, and it's been difficult to figure out which strings correspond to which. For me, it isn't obvious if I need to keep my component named the same as my file, etc.

Even if the names are silly sounding, it would be great if they were very different so I could see which things needed to match up easily.

**Comment by wchargin** [comment]
To get you unblocked:
  * You can call your directory whatever you want.
  * The `path` attribute of the build rule, which I believe @jart calls the "webfiles path," is what's used for import statements. So `path = "/foo"` induces `<link rel="import" href="foo/my-file.html">` and `import {MyClass} from './foo/myScript.js'`.
  * You can give the component any name that you want, as long as it has a hyphen (Polymer requirement to distinguish web components from built-in components), but I'd recommend that you call it `*-dashboard`, so `beholder-dashboard`. (I.e., I'd prefer that `my-plugin` be renamed to `my-time-dashboard` or similar.)
  * You can call your HTML file whatever you want. (See also webfiles path above.)
  * The key in the `TENSORBOARD_PLUGINS` object is an arbitrary string that will be used as the *tab name* in TensorBoard.
  * The value in the `TENSORBOARD_PLUGINS` object must be the name of your Polymer component.

In general, I think that the names in the example repository should, unsurprisingly, follow the patterns of real names. (To this effect, the component should be renamed, as described above.) It seems that an explanatory comment in each of these places would satisfy your use case, yes?

**Comment by chrisranderson** [comment]
Yep, comments would be great. My code matched what you said, but my `main.py` wasn't right. I was using the one from the TensorBoard repo, not from the example. I've now got it working using this structure. Thanks to both of you!
